aws/build.gradle

```diff
@@ -93,7 +93,6 @@ task deploy(type: AWSLambdaInvokeTask) {
   "experiments": "${ext.getParameterValue('beamExperiments') ?: ext.getParameterValue(getCurrentGitBranch() + '.experiments')}",
   "execute_class": "${ext.getParameterValue('executeClass')}",
   "execute_args": "${ext.getParameterValue('executeArgs')}",
-  "forced_max_ram": "${ext.getParameterValue('forcedMaxRAM')}",
```

Comment:
why remove this parameter?

Reply:
this parameter was taking default as 'null' . we don't need to use its value as null. so completely removed it. If someone need this param then they need to provide some valid value